### PR TITLE
Extra Lines at end of ICAL file throws parser error.

### PR DESCRIPTION
**Pull request by kewisch**
Parser should probably discard extra lines after the end of the root node. Its easy to work around by calling `rawICALString.trim()`

### CODE REVIEW

**Comment by kewisch** [comment]
So this will work for now, I don't think it will hurt performance since the regex can stop testing as soon as something other than a whitespace is found.

Ideally I'd like to have a sort of ErrorCollector. While parsing, you can add warnings and errors to it, and the ErrorCollector is configurable with things like warningsAsErrors or continuing despite errors. Thats something for a different issue though.

File: test/mocha/ical_test.js [hunk 1]
```diff
@@ -34,4 +35,28 @@ suite('ical/parser', function() {
...
+
+  test('Blank line at end', function() {
+    // This sample contains a blank line at the end, it should still parse
+    var result = ICAL.parse(blankLineEndICS);
+  });
+
+  test('Blank line within', function() {
+    // This sample contains a blank line at the end, it should throw an error
+    var error, result;
+    try {
```
Comment (lightsofapollo) [inline test/mocha/ical_test.js:53]: This is a fine way to test it but you can also use this method http://chaijs.com/api/assert/#throws-section

``` js


assert.throws(function() {
   ICAL.parse(blankLineMidICS);
}, /messageRegex/)

```

**Comment by lightsofapollo** [comment]
@kewisch r=lightsofapollo

I mentioned an alternative in the test file but the current method is fine if a little longer.

**Comment by kewisch** [comment]
I've changed the code to follow your review comment on the master, changeset 743f4bb7692b4ec9c874dd532cffe037f75fcbb0

**Pull request merged**